common/config/rush/pnpm-lock.yaml

```diff
@@ -2853,6 +2852,188 @@ packages:
       '@bentley/bentleyjs-core': ^2.6.4
     resolution:
       integrity: sha512-SbYyH6vsLi+zdEBMvyK3dFStQdVUpSkR7zusslGVmxOqaXVxM45Gv0esy8KiQh94VYtrJ2jK2MsNE9qcBmGSRQ==
+  /@bentley/itwin-error-handling-react/3.1.1_@bentley+ui-core@2.6.4:
```

Comment:
this file should not change if you haven't added/updated any dependencies?

Reply:
not quite sure why. I'll revert